src/controllers/profile.js

```diff
@@ -1,46 +1,17 @@
-import { dirname, resolve } from 'node:path';
-import { fileURLToPath } from 'node:url';
-
+import ProfileParser from '@wfcd/profile-parser';
 import express from 'express';
-import ArsenalParser from '@wfcd/arsenal-parser';
-import flatCache from 'flat-cache';
 
-import { noResult, cache } from '../lib/utilities.js';
+import { cache, noResult } from '../lib/utilities.js';
 
 const router = express.Router({ strict: true });
-
-const WF_ARSENAL_ID = 'ud1zj704c0eb1s553jbkayvqxjft97';
-const WF_ARSENAL_API = 'https://content.warframe.com/dynamic/twitch/getActiveLoadout.php';
-let token;
-
-const dirName = dirname(fileURLToPath(import.meta.url));
-
-router.use((req, res, next) => {
-  const tokenCache = flatCache.load('.twitch', resolve(dirName, '../../'));
-  token = tokenCache.getKey('token');
-  next();
-});
+const WF_PROFILE_API = 'https://content.warframe.com/dynamic/getProfileViewingData.php';
```

Comment:
this goes in src/lib/settings.js.... i know i didn't do it before, but please do now.

also, we'd talked about moving arsenal to another (still-existing) route, this just removes it

Reply:
sorry, i probably wasn't clear either

Reply:
No worries. Got to do the other thing I wanted to do 😅